openlibrary/templates/covers/change.html

```diff
@@ -29,7 +29,7 @@
 
 <div class="$size sansserif manageCoversContainer hidden--nojs">
   <a aria-controls="addImage-$imagesId" class="coverPop dialog--open">
```

Comment:
This shouldn't need to be disabled for non-js clients; we can add a fallback so that the iframe opens in a new tab by adding the href.

```suggestion
<div class="$size sansserif manageCoversContainer">
  <a aria-controls="addImage-$imagesId" class="coverPop dialog--open" href="$(manage_url if doc.get_cover() else add_url)">
```

Sorry I know this was here before you :P But noting since it adds context for my next comment...